apps/src/applab/designElements/predictPanel.jsx

```diff
@@ -0,0 +1,200 @@
+import $ from 'jquery';
+import PropTypes from 'prop-types';
+import React from 'react';
+import PropertyRow from './PropertyRow';
+import BooleanPropertyRow from './BooleanPropertyRow';
+import ColorPickerPropertyRow from './ColorPickerPropertyRow';
+import ZOrderRow from './ZOrderRow';
+import EventHeaderRow from './EventHeaderRow';
+import EventRow from './EventRow';
+import BorderProperties from './BorderProperties';
+import themeValues from '../themeValues';
+import * as elementUtils from './elementUtils';
+import designMode from '../designMode';
+import elementLibrary from './library';
+import i18n from '@cdo/applab/locale';
+
+class PredictPanelProperties extends React.Component {
+  static propTypes = {
+    element: PropTypes.instanceOf(HTMLElement).isRequired,
+    handleChange: PropTypes.func.isRequired,
+    onDepthChange: PropTypes.func.isRequired,
+    mlModelDetails: PropTypes.object
+  };
+
+  render() {
+    const element = this.props.element;
+
+    return (
+      <div id="propertyRowContainer">
+        <PropertyRow
+          desc={'id'}
+          initialValue={elementUtils.getId(element)}
+          handleChange={this.props.handleChange.bind(this, 'id')}
+          isIdRow
+        />
+        <PropertyRow
+          desc={'model id'}
+          initialValue={elementUtils.getId(element)}
+          handleChange={this.props.handleChange.bind(this, 'model id')}
```

Comment:
You probably want to add a switch case for this change in the [updateProperty function](https://github.com/code-dot-org/code-dot-org/blob/bf3b496b5f212b9d737d876de50a71c21c751109/apps/src/applab/designMode.js#L228). That function is also another option for where to make the API call or somewhere in [here](https://github.com/code-dot-org/code-dot-org/blob/bf3b496b5f212b9d737d876de50a71c21c751109/apps/src/applab/designMode.js#L214) which is the handleChange that is being referenced here